gr-utils/python/modtool/gr-newmod/docs/doxygen/Doxyfile.swig_doc.in

```diff
@@ -121,7 +121,7 @@ INLINE_INHERITED_MEMB  = NO
 # path before files name in the file list and in the header files. If set
 # to NO the shortest path that makes the file name unique will be used.
 
-FULL_PATH_NAMES        = YES
+FULL_PATH_NAMES        = NO
```

Comment:
What's the result of this change?

Reply:
That the absolute build path doesn't end up in docs, meaning the built files are not dependent on where they were built.